cumulusci/core/source/github.py

```diff
@@ -124,18 +125,12 @@ def resolve(self):
         self.description = ref[6:] if ref.startswith("heads/") else ref
         self.commit = self.repo.ref(ref).object.sha
 
-    def fetch(self, path=None):
+    def fetch(self):
         """Fetch the archive of the specified commit and construct its project config."""
-        # To do: copy this from a shared cache
-        if path is None:
-            path = (
-                self.project_config.cache_dir
-                / "projects"
-                / self.repo_name
-                / self.commit
-            )
-        if not path.exists():
-            path.mkdir(parents=True)
+        # fs.path, which we use in open_cache, doesn't work with Pathlib path objects
+        with self.project_config.open_cache(
+            fs.path.join("projects", self.repo_name, self.commit)
```

Comment:
I don't think this comment is accurate. I see `open_cache` is using `project_config.cache_dir`, which is a `Path` instance, and the function signature for `open_fs_resource` accepts `Union[str, Path, "FSResource"]`.

Reply:
I'm trying to remember what breakage I found - I think it's actually that it doesn't work with Pathlib paths where one or more path components do not yet exist (?). I'll remove the comment in any case.